podstawowe_operacje/Installment/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿var price = (decimal)-1;
+
+while (price < 0)
+{
+    Console.Write("Podaj cenę towaru: ");
+    price = decimal.Parse(Console.ReadLine());
+}
+
+var installmentsCount = 1;
+
+while (installmentsCount < 2 || installmentsCount > 24)
+{
+    Console.Write("Podaj liczbę rat (od 2 do 24): ");
+    installmentsCount = int.Parse(Console.ReadLine());
+}
+
+var installmentPayDay = -1;
+while (installmentPayDay < 0)
+{
+    Console.Write("Podaj dzień płatności raty (dzień dla każdego miesiąca, w którym ma być płatna rata): ");
+    installmentPayDay = int.Parse(Console.ReadLine());
+}
+
+Console.Write("Podaj wpłatę klienta (od 0 zł): ");
+var payment = decimal.Parse(Console.ReadLine());
+
+
+var currentDate = DateTime.Now;
+
+var basePayDate = new DateTime(
+    currentDate.Year,
+    currentDate.Month,
+    installmentPayDay
+    );
+
+var payDates = CalculatePayDates(installmentsCount, basePayDate);
+
+var interest = CalculateInterest(installmentPayDay);
+
+var amountOfCredit = price + price * interest;
+
+var instellmentAmount = (amountOfCredit / installmentsCount);
+
+Console.WriteLine(" kwota wypłaty | data płatności | kwota kredytu | odsetki");
+foreach (var date in payDates)
+{
+    amountOfCredit = amountOfCredit - instellmentAmount;
+    Console.WriteLine($"{Math.Round(instellmentAmount,2)} | " +
+        $"{date.ToString("dd'-'MMMM'-'yyyy")} " +
+        $"| {Math.Round(amountOfCredit, 2)} | {interest}");
+}
+
+static decimal CalculateInterest(decimal installmentPayDay)
+{
+    if (installmentPayDay <= 8)
+    {
+        return 0;
+    }
+    if (installmentPayDay >= 9 && installmentPayDay <= 12)
+    {
+        return (decimal)0.05;
```

Comment:
```suggestion
        return 0.05m;
```

Reply:
Dodałem we wcześniejszym commicie.

Reply:
[01891c6](https://github.com/broniuu/Wst-pDoProgramowania/pull/1/commits/01891c6a922b5dd2ce11edf32aa9e9a65708e03f)